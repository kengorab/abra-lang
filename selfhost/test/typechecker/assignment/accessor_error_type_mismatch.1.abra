type Foo { a: Int }

val f = Foo(a: 123)
f.a = "abcd"