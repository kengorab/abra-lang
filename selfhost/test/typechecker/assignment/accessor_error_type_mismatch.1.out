Error at %FILE_NAME%:4:7
Type mismatch
  |  f.a = "abcd"
           ^
Expected: Int
but instead found: String
